Allow optional actor ObjectRef value in node interaction calls (#14505)
minetest/minetest: doc/lua_api.md

      lsystem_decoration_type = true,
      -- Overrideable pointing range using the itemstack meta key `"range"` (5.9.0)
      item_meta_range = true,

>       -- Allow passing in an optional "actor" ObjectRef into node interaction calls (5.9.0)

Reviewer: This should mention the affected functions.

Author: Resolved in 391f46c62105d536b0fef43a62767c10b95a21a1

      lsystem_decoration_type = true,
      -- Overrideable pointing range using the itemstack meta key `"range"` (5.9.0)
      item_meta_range = true,

>       -- Allow passing in an optional "actor" ObjectRef into the following functions: (5.9.0)
>       -- minetest.place_node, minetest.dig_node, minetest.punch_node

Reviewer: ```suggestion
      -- Allow passing an optional "actor" ObjectRef to the following functions:
      -- minetest.place_node, minetest.dig_node, minetest.punch_node (5.9.0)
```
